File: challenges/3-area-trapezio/python/jbernanos/trapezio.py
try:
    def dados():
        altura = float(input('Informe o valor da altura do trapézio: '))
        base_maior = float(input('Informe o valor da base maior: '))
        base_menor = float(input('Informe o valor da base menor: '))
        area(altura, base_maior, base_menor)
        return altura, base_maior, base_menor

    def area(a, bma, bme):
        calculo = (bma + bme) * a / 2
        print(f'A área de um trapézio com altura {a}, base maior {bma} e base menor {bme} é: {calculo}!')

    dados()

except ValueError:
    print('Dado não aceito por não ser um número.')

File: challenges/3-area-trapezio/c/paclucas/trapezio.c
#include <stdio.h>

int main()
{
  float baseMaior, baseMenor, altura;
  printf("\nValor base maior: ");
  scanf("%f", &baseMaior);

  printf("\nValor base menor: ");
  scanf("%f", &baseMenor);

  printf("\nValor altura: ");
  scanf("%f", &altura);

  float res = (baseMaior + baseMenor) * altura / 2;
  printf("\nA área do trapézio é: %.1f", res);

  printf("\n^._.^\n");

  return 0;
}